demos/typescript-app/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+      "outDir": "./dist/",
+      "noImplicitAny": true,
+      "module": "es6",
+      "target": "es5",
+      "jsx": "react",
+      "allowJs": true,
+      "moduleResolution": "node"
```

Comment:
Please refer to the root tsconfig setting and update it.
It seems there are some unnecessary parameters. (ex. jsx)

Reply:
I will refer and update it